modules/graphql_1/main.tf

```diff
@@ -145,7 +177,7 @@ resource "aws_ecs_task_definition" "gql_task" {
 [
   {
     "name": "orca-gql",
-    "image": "ghcr.io/nasa/cumulus-orca/graphql:0.0.22",
+    "image": "ghcr.io/nasa/cumulus-orca/graphql:0.0.24",
```

Comment:
we have to remember to update this version every time a new version is released

Reply:
Sadly true. I've documented everything in the README.